.github/workflows/pull-request.yml

```diff
@@ -78,6 +79,15 @@ jobs:
       run: |
         exit 1
 
+    - name: Test assign-user action without role assignment
+      uses: ./assign-user
+      with:
+        user-name: ${{ env.WF_USERNAME }}
+        password-secret: ${{ secrets.PASSWORD_PPDEVTOOLS }}
+        environment: ${{ steps.create-environment.outputs.environment-url }}
+        user: ${{ env.WF_USERNAME}}
```

Comment:
should we use a separate user?

Reply:
We can grab the ID of another user to test with to ensure the API call to sync is fully functional at this level.

Reply:
This has been updated, per your request.